doc/en/components/themes/palettes.md

```diff
@@ -0,0 +1,227 @@
+
+On top of the aforementioned colors, we also include **Level AAA** [WCAG](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html) compliant `contrast` colors for each color variant. This means that you can safely use the corresponding `contrast` color variants as foreground colors for the base color variant.
+
+> [!NOTE]
+> Contrast colors are generated at build-time therefore overriding the CSS variables will not update the corresponding contrast colors.
+
+Here's an excerpt of the `primary` color as declared in the Light Bootstrap Palette:
+
+```css
+:root {
+  --igc-primary-h: 216deg;
+  --igc-primary-s: 98%;
+  --igc-primary-l: 52%;
+  --igc-primary-a: 1;
+  --igc-primary-50: hsla(
+                      var(--igc-primary-h),
+                      calc(var(--igc-primary-s) * 1.23),
+                      calc(var(--igc-primary-l) * 1.78),
+                      var(--igc-primary-a)
+                    );
+  --igc-primary-100: hsla(
+                      var(--igc-primary-h),
+                      calc(var(--igc-primary-s) / 1.25),
+                      calc(var(--igc-primary-l) * 1.66),
+                      var(--igc-primary-a)
+                    );
+  --igc-primary-200: hsla(
+                      var(--igc-primary-h),
+                      calc(var(--igc-primary-s) * .64),
+                      calc(var(--igc-primary-l) * 1.43),
+                      var(--igc-primary-a)
+                    );
+  // ...
+  --igc-primary-50-contrast: black;
+  --igc-primary-100-contrast: black;
+  --igc-primary-200-contrast: black;
+  // ...
+}
+```
+
+All color variants are derived from 4 base variables - `--igc-primary-h`, `--igc-primary-s`, `--igc-primary-l`, and `--igc-primary-a`. Each one of these variables holds the [HSLA](https://drafts.csswg.org/css-color/#the-hsl-notation) parts for a single color. HSLA stands for `hue`, `saturation`, `lightness`, and `alpha`. It's another color scheme used to describe colors. We decided to use this approach as it allows us to modify all variants of the `primary`, `secondary` and other colors at runtime.
+
+## Defining Palettes
+
+If you wanted to change the color variants for a color from the palette, you can do so by overriding its HSLA values in your stylesheet. For instance, changing the primary colors is as easy as writing:
+
+```css
+/* The HSLA representation of orange (#ffa500) */
+/* hsla(38.8, 100%, 50%, 1); */
+:root {
+  --igc-primary-h: 38.8deg; 
+  --igc-primary-s: 100%; 
+  --igc-primary-l: 50%; 
+  --igc-primary-a: 1; 
+}
+```
+
+This will automatically update all the other primary variants.
+
+You will notice that color variants for each color are monochromatic. This is because all color variants are generated from the HSLA variables. You can override individual colors only using any color scheme:
+
+```css
+:root {
+  --igc-primary-600: darkorange;
+}
+```
```

Comment:
This has to remain in the HSLA color space, unfortunately. Further, it has to be in the format of list of values for h, s, l, and a.

```suggestion
```css
:root {
  --igc-primary-600: 32.9deg ,100%, 50%, 1;
}
```
```